tests/contenttypes_tests/test_fields.py

```diff
@@ -33,3 +33,11 @@ def test_value_to_string(self):
         answer2 = Answer.objects.create(question=question)
         result = json.loads(Question.answer_set.field.value_to_string(question))
         self.assertCountEqual(result, [answer1.pk, answer2.pk])
+
+    def test_filter_on_related_proxy_model(self):
+        question = Question.objects.create(text='test')
+        Answer.objects.create(question=question)
+        self.assertEqual(
```

Comment:
A single line looks okay here. Our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) allows lines up to 119 characters if it helps readability. You could use `get()` rather than `first()` to make it a bit shorter.